products/red-hat-openshift.md

```diff
@@ -0,0 +1,113 @@
+    support: 2021-03-24
+    eol: 2022-10-27
+    latest: "4.6.62"
+    latestReleaseDate: 2022-11-10
+    extendedSupport: false
+-   releaseCycle: "4.5"
+    releaseDate: 2020-07-13
+    support: 2020-11-27
+    eol: 2021-07-27
+    latest: "4.5.41"
+    latestReleaseDate: 2021-06-30
+    extendedSupport: false
+-   releaseCycle: "4.4"
+    releaseDate: 2020-05-05
+    support: 2020-08-13
+    eol: 2021-02-24
+    latest: "4.4.33"
+    latestReleaseDate: 2021-02-02
+    extendedSupport: false
+-   releaseCycle: "4.3"
+    releaseDate: 2020-01-23
+    support: 2020-06-05
+    eol: 2020-10-27
+    latest: "4.3.40"
+    latestReleaseDate: 2020-10-20
+    extendedSupport: false
+-   releaseCycle: "4.2"
+    releaseDate: 2019-10-16
+    support: 2020-02-23
+    eol: 2020-07-13
+    latest: "4.2.36"
+    latestReleaseDate: 2020-07-01
+    extendedSupport: false
+-   releaseCycle: "4.1"
+    releaseDate: 2019-06-04
+    support: 2019-11-16
+    eol: 2020-05-05
+    latest: "4.1.41"
+    latestReleaseDate: 2020-04-22
+    extendedSupport: false
+
+---
+
+>[OpenShift](https://www.redhat.com/en/technologies/cloud-computing/openshift) is a family of containerization software products developed by Red Hat. Its flagship product is the OpenShift Container Platform — a hybrid cloud platform as a service built around Linux containers orchestrated and managed by Kubernetes on a foundation of Red Hat Enterprise Linux.
+
+Red Hat provides three different phases of support:
+
+* **Full Support**: This phase begins at the GA/release of the minor version and ends after a 6 month period OR 90 days after the GA of the superseding minor release, whichever is later. During the Full Support Phase, qualified Critical and Important Security Advisories ([RHSAs][DEFINITION]) and Urgent and Selected High Priority Bug Fix Advisories ([RHBAs][DEFINITION]) will be released as they become available; all other available fix and qualified patches may be released via periodic updates.
+* **Maintenance Support**: This phase commences after the Full Support phase for the respective minor version and ends at 18 months after GA. During the Maintenance Support phase, qualified Critical and Important Security Advisories (RHSAs) and Urgent and Selected High Priority Bug Fix Advisories (RHBAs) may be released as they become available. Other Bug Fix (and Enhancement ([RHEA][DEFINITION]) Advisories may be released at Red Hat’s discretion, but should not be expected.
+* **Extended Update Support**: All even numbered minor releases (eg. 4.8, 4.10, 4.12) are Extended Update Support (EUS) releases. Additionaly to the full support and maintenance support phases, which also apply to EUS releases, upgrades between EUS releases will cause less reboots of nodes. Commencing with OpenShift Container Platform 4.12, Red Hat will additionally include a six month Extended Update Support (EUS) phase for all EUS releases. The EUS phase will follow the maintenance phase for the given release.
```

Comment:
> Commencing with OpenShift Container Platform 4.12, Red Hat will additionally include a six month Extended Update Support (EUS) phase for all EUS releases.

Confused about this wording. Should this just be _all releases_, because its unclear what a EUS release is.

Reply:
Well, every release with an even minor version is an EUS release. In the past, the only benefit using these was, that it's a little bit more efficient (less reboots) to upgrade between those versions. From version 4.12+ (this is also an EUS release) on, there is another benefit: additional 6 months of support. Which results in a total support time of 2 years for EUS releases.

All "normal" releases (eg. 4.11, 4.13, 4.15) have a total support time of 18 month (~6 months or more full support and the rest is maintenance support).

Reply:
@captn3m0 Is it better now?^^